AppLibraryEnabler.xm

```diff
@@ -160,29 +160,12 @@
 }
 %end
 
-%hook SBIconView
-- (bool)allowsAccessoryView {
-	bool origValue = %orig;
-	if ( [[self _viewControllerForAncestor] isKindOfClass:%c( SBHIconLibraryTableViewController )] || [[self _viewControllerForAncestor] isKindOfClass:%c( SBHLibraryCategoryIconViewController )] || [[self _viewControllerForAncestor] isKindOfClass:%c( SBHLibraryPodCategoryFolderController )] ) {
-		NSMutableDictionary *defaults = [NSMutableDictionary dictionaryWithContentsOfFile:[NSString stringWithFormat:@"%@/Library/Preferences/com.apple.springboard.plist", NSHomeDirectory()]];
-		bool sbHomeScreenShowsBadgesInAppLibrary = [[defaults objectForKey:@"SBHomeScreenShowsBadgesInAppLibrary"] boolValue];
-		return sbHomeScreenShowsBadgesInAppLibrary;
-	}
-	return origValue;
+extern "C" bool _os_feature_enabled_impl(const char *domain, const char *feature);
+%hookf(bool, _os_feature_enabled_impl, const char *domain, const char *feature) {
+    if (strcmp(domain, "SpringBoard") == 0 && strcmp(feature, "Dewey") == 0)
```

Comment:
Protip: check out `/System/Library/FeatureFlags/Global.plist`

Reply:
This file does not exist in .ipsw (or it's iOS13+)? - just checked in 12.5.3.ipsw and it wasn't there

If it exist then can it be used to enable pip on iOS12?

Reply:
Just checked in 14.6.ipsw - it exists there :)